content/en/docs/concepts/cluster-administration/compatibility-version.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Compatibility Version For Kubernetes Control Plane Components
+reviewers:
+- jpbetz
+- siyuanfoundation
+content_type: concept
+weight: 70
+---
+
+<!-- overview -->
+
+Since release v1.32, we introduced configurable version compatibility and emulation options to Kubernetes control plane components to make upgrades safer by providing more control and increasing the granularity of steps available to cluster administrators.
+
+<!-- body -->
+
+## Emulated Version
+
+The emulation option is set by the `--emulated-version` flag of control plane components. It allows the component to emulate the behavior (APIs, features, ...) of an earlier version of Kubernetes.
+
+When used, the capabilities available will match the emulated version: 
```

Comment:
nit: is capabilities a k8s term cluster admins are familiar with? It might make sense to explicity state that --emulated-version specifies emulation of APIs & feature/feature-gates but I think the current text is ok as well.

Reply:
hm, do we have enough coverage in the previous sentence of 

"
It allows the component to emulate the behavior (APIs, features, ...) of an earlier version of Kubernetes." 

or should it be said more explicitly?